Wyjątki dzielimy na KONTROLOWANE które trzeba obsłużyć i NIEKONTROLOWANE które można, ale nie trzeba obsłużyć.
Wyjątki NIEKONTROLOWANE dziedziczą po klasie RuntimeException
Wyjątki rzucamy korzystając z instrukcji THROW, jeśli rzucamy wyjątek kontrolowany musimy dodatkowo w sygnaturze metody/konstruktora zadeklarować wyjątek używając słowa THROWS <KlasaWyjatku>
Do bloku try-catch możemy dodać dodatkowo blok FINALLY, który wykona się zawsze, niezależnie, czy w bloku try wyjątek wystąpił, czy nie